edx_proctoring/static/proctoring/js/views/proctored_exam_onboarding_view.js

```diff
@@ -18,7 +18,7 @@ edx = edx || {};
     onboardingStatusReadableFormat = {
         not_started: gettext('Not Started'),
         setup_started: gettext('Setup Started'),
-        proctoring_started: gettext('Proctoring Started'),
+        onboarding_started: gettext('Proctoring Started'),
```

Comment:
Just missed one string here, but otherwise LGTM

Reply:
Thank you!